packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -33,11 +34,15 @@ const OdysseyProvider = ({
   children,
   nonce,
   themeOverride,
+  tokenOverride,
   languageCode,
   translationOverrides,
 }: OdysseyProviderProps) => (
   <OdysseyCacheProvider nonce={nonce}>
-    <OdysseyThemeProvider themeOverride={themeOverride}>
+    <OdysseyThemeProvider
+      themeOverride={themeOverride}
+      tokenOverride={tokenOverride}
+    >
```

Comment:
**NON-BLOCKING** We no longer need `themeOverride` with the new system, but it's fine to leave this here for now. We'll wanna make a ticket to remove it before v1.0's release.

Reply:
https://oktainc.atlassian.net/browse/OKTA-612336